include/swift/AST/Types.h

```diff
@@ -289,7 +289,7 @@ class alignas(1 << TypeAlignInBits) TypeBase {
 
     /// Extra information which affects how the function is called, like
     /// regparm and the calling convention.
-    unsigned ExtInfo : 7;
+    unsigned ExtInfo : 7; enum { NumExtInfoBits = 7 };
```

Comment:
Could you actually use the magic number here? :)

Otherwise this looks good, thanks!

Reply:
Unless I'm missing something, that would require moving and renaming the two `ExtInfo` classes so that they could be declared and defined before `TypeBase`, right? Is that what you want done?

Reply:
I'm asking for something much trivial than that; I am asking you to write:

  enum { NumExtInfoBits = 7 };
  unsigned ExtInfo : NumExtInfoBits;

Reply:
Oh, sure. Done. I'll push the change in a second. :-)

FWIW – I thought you might be asking for the two `ExtInfo` types to be more like `RecursiveTypeProperties`, where `TypeBase` uses a constant defined by `RecursiveTypeProperties` to allocate the number of reserved bits. That would have required moving and renaming the two `ExtInfo` types ahead of `TypeBase`.